eng/common/scripts/Package-Properties.ps1

```diff
@@ -110,7 +99,7 @@ function Get-PkgProperties
 # Takes ServiceName and Repo Root Directory
 # Returns important properties for each package in the specified service, or entire repo if the serviceName is not specified
 # Returns an Table of service key to array values of PS Object with properties @ { pkgName, pkgVersion, pkgDirectoryPath, pkgReadMePath, pkgChangeLogPath }
-function Get-AllPkgProperties ([string]$ServiceDirectory = $null)
+function Get-AllPkgProperties ([string]$ServiceDirectory = $null, [bool]$includeMgmt = $true)
```

Comment:
For java I thought the ci.mgmt.yml file didn't list any artifacts? If that is the case then they will be excluded automatically because they aren't listed.